src/template.json

```diff
@@ -38,12 +38,23 @@
     }
   ],
   "provisioners": [
+    {
+      "script": "shutdown-apt-services.sh",
+      "type": "shell"
+    },
     {
       "ansible_env_vars": [
         "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'"
       ],
       "playbook_file": "playbook.yaml",
       "type": "ansible"
+    },
+    {
+      "inline": [
+        "sudo systemctl enable apt-daily.timer",
```

Comment:
this does not guarantee execution order with respect to the other shell provisioner defined in this file if that's important to you.. https://packer.io/docs/provisioners/shell.html#scripts

Reply:
I'm not sure if this is important, this is just copied from the webpage you sent me. https://codeinthehole.com/tips/avoiding-package-lockout-in-ubuntu-1804/